great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py

```diff
@@ -3659,23 +3668,26 @@ def _create_df_for_charting(
         df = pd.concat([df, batch_identifier_df], axis=1)
 
         if plot_mode == PlotMode.DIAGNOSTIC:
-            for kwarg_name in expectation_configuration.kwargs:
-                # column name was already retrieved from the domain
-                if isinstance(expectation_configuration.kwargs[kwarg_name], dict):
-                    for key, value in expectation_configuration.kwargs[
-                        kwarg_name
-                    ].items():
-                        if isinstance(value, list):
-                            df[key] = [value for _ in df.index]
-                        else:
-                            df[key] = value
-
-                elif isinstance(expectation_configuration.kwargs[kwarg_name], list):
-                    df[kwarg_name] = [
-                        expectation_configuration.kwargs[kwarg_name] for _ in df.index
-                    ]
-                else:
-                    df[kwarg_name] = expectation_configuration.kwargs[kwarg_name]
+            if expectation_configuration is not None:
```

Comment:
Since we are in diagnostic mode, if there is no expectation configuration associated with this metric, return None.

Reply:
Just a suggestion - could we return an empty dataframe instead of None to make the return type consistent?